5_0311/Program.cs

```diff
@@ -23,26 +23,77 @@ static void Main(string[] args)
             //Stars2();
 
             Stars3();
+            Stars4();
 
 
             
 
         }
-
         // TODO
         /// <summary>
         /// pt n = 4 se va afisa:
-        /// ********
-        /// ***  ***
-        /// **    **
         /// *      *
         /// **    **
         /// ***  ***
         /// ********
-        /// </summary>
-        private static void Stars3()
+        /// <summary>
+
+        private static void Stars4()
```

Comment:
Nu era nevoie de metoda Stars4. Probabil ca e bine ce ai facut doar ca ai impartit in 2 metode ceea ce trebuia facut intr-una singura.